### PR TITLE
Note name typein, about box tweak

### PR DESCRIPTION
**Pull request by mkruselj**
* note names for now only work on split key and keytrack root
* both sharps and flats are supported
* about box has the Surge github link colored now (aboutbox.link skin color)

### CODE REVIEW

File: src/common/Parameter.cpp [hunk 1]
```diff
@@ -1764,9 +1764,13 @@ bool Parameter::set_value_from_string( std::string s )
       {
          ni = std::stoi(c);
       }
-      catch (std::invalid_argument const& e)
+      catch (std::invalid_argument)
       {
-         ni = val_min.i - 1;   // throw things out of range on invalid input
+         ni = val_min.i - 1;   // set value of ni out of range on invalid input
+      }
```
Comment (baconpaul) [inline src/common/Parameter.cpp:1770]: You would want the const on the second exception also

Comment (baconpaul) [reply src/common/Parameter.cpp:1770]: And crazily you want const std::invalid_argument &e, not std::invalid_argument const& e, which means something different. Const is hard.

File: src/common/Parameter.cpp [hunk 2]
```diff
@@ -1793,7 +1870,7 @@ bool Parameter::set_value_from_string( std::string s )
       return false;
    }
 
-   auto nv = std::atof(c);
+   float nv = std::atof(c);
```
Comment (baconpaul) [inline src/common/Parameter.cpp:1873]: curious why you changed auto to float here?

Comment (mkruselj) [reply src/common/Parameter.cpp:1873]: I had the same try-catch as for ints but then reverted it and forgot to reinstate `auto`.

**Pull request merged**